pkg/cmd/chat/reaction/reaction.go

```diff
@@ -0,0 +1,133 @@
+package reaction
+
+import (
+	stream_chat "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getCmd(),
+		sendCmd(),
+		deleteCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-reactions [message-id]",
+		Short: "Get reactions for a message",
+		Example: heredoc.Doc(`
+			# Get reactions for a [08f64828-3bba-42bd-8430-c26a3634ee5c] message
+			$ stream-cli chat get-reactions 08f64828-3bba-42bd-8430-c26a3634ee5c --output-format json
+		`),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			msgId := args[0]
+
+			resp, err := c.Channel("", "").GetReactions(cmd.Context(), msgId, nil)
+			if err != nil {
+				return err
+			}
```

Comment:
this is weird, need to fix this in sdk

Reply:
also prefer `msgID`, it's more go idiomatic

Reply:
where?

Reply:
forgot to save it 😆

Reply:
Plus: https://github.com/GetStream/stream-chat-go/pull/215